ed448-goldilocks/src/field/element.rs

```diff
@@ -265,10 +265,7 @@ impl FieldElement {
     /// Inverts a field element
     /// Previous chain length: 462, new length 460
```

Comment:
I assume this comment was referring to an addition chain at one point but perhaps could be removed now?

Reply:
Done!